zone/raids.cpp

```diff
@@ -1741,6 +1780,18 @@ void Raid::VerifyRaid()
 			else {
 				m.member = nullptr;
 			}
+			
+			for (int i = 0; i < MAX_NO_RAID_MAIN_MARKERS; i++) {
+				if (m.main_marker == i + 1) {
```

Comment:
Is the + 1 right here? Just making sure it's intentional

Reply:
It is and thanks for confirming.  The client uses 1, 2, 3 for numbering the markers and assisters, and use 0 to represent 'not an assister or marker' so it is adjusted here and in a few other places as well.